src/lf_workflow_dash/github_request.py

```diff
@@ -25,28 +50,39 @@ def update_workflow_status(workflow_elem, token):
     response = requests.request("GET", request_url, headers=headers, data=payload, timeout=15)
     status_code = response.status_code
     conclusion = "pending"
+    conclusion_time = ""
+    # TODO set is_stale
 
     # Process data
     if status_code == 200:  # API was successful
         response_json = response.json()
         if len(response_json["workflow_runs"]) == 0:  # workflow has no runs
             conclusion = "not yet run"
+        
         else:
             last_run = response_json["workflow_runs"][0]
 
             # Get the workflow conclusion ("success", "failure", etc)
             conclusion = last_run["conclusion"]
 
+            # Get the time this workflow concluded (in New York time)
+            conclusion_time = get_conclusion_time(last_run)
+
             # Check if the workflow is currently being executed
             if conclusion is None:
                 # try next most recent
                 if len(response_json["workflow_runs"]) > 1:
                     last_run = response_json["workflow_runs"][1]
                     conclusion = last_run["conclusion"]
+                    conclusion_time = get_conclusion_time(last_run)
                 else:
                     conclusion = "pending"
+                    conclusion_time = ""
+
+            # TODO set is_stale
```

Comment:
It makes me very sad to see TODOs getting pylint warnings. Opened https://github.com/lincc-frameworks/python-project-template/issues/341 to allow a comment like `# TODO(#16) set is_stale`.